StitchCoreServicesHttp-iOS/StitchCoreServicesHttp-iOS/StitchCoreServicesHttp_iOS.h

```diff
@@ -0,0 +1,11 @@
+#import <UIKit/UIKit.h>
+
+//! Project version number for StitchCoreServicesHttp_iOS.
+FOUNDATION_EXPORT double StitchCoreServicesHttp_iOSVersionNumber;
+
+//! Project version string for StitchCoreServicesHttp_iOS.
+FOUNDATION_EXPORT const unsigned char StitchCoreServicesHttp_iOSVersionString[];
+
+// In this header, you should import all the public headers of your framework using statements like #import <StitchCoreServicesHttp_iOS/PublicHeader.h>
```

Comment:
[nit] Can remove these newlines. We can probably start removing these headers as well.

Reply:
Removed the newlines. Are these headers for the sake of Objective-C compatibility? Happy to remove them if we are definitely not supporting objc